Main.gd

```diff
@@ -264,6 +264,7 @@ static func _get_center(parent: Node2D) -> Vector2:
 
 func _on_tab_tab_changed(tab):
 	SelectedListManager.clear_list()
+	SelectedListManager.close_details_panel()
```

Comment:
If clearing the selection list already closes the details panel, and changing tabs clears the selection list. You don't have to explicitly do this